exporter.go

```diff
@@ -27,6 +27,16 @@ func newMQTTExporter() *mqttExporter {
 	options := mqtt.NewClientOptions()
 	log.Infof("Connecting to %v", *brokerAddress)
 	options.AddBroker(*brokerAddress)
+	options.SetAutoReconnect(true)
```

Comment:
Some more config (setAutorReconnect) seems to have snuck into this commit.
The default for Autoreconnect is already true, so not entirely sure this is required to be added.

Reply:
Sorry, removed.